headwindsim/SU95X
# Change Payload Balance

Fixes #[issue_no]

## Summary of Changes

Trying to match this document from FCOM.
The pax now consist of 5 stations which are currently causing some issue with the EFB.

## Screenshots (if necessary)



## References

![image](https://user-images.githubusercontent.com/19798208/216749877-25cea3a8-88e8-41b6-8ec6-5780dcd708d8.png)

## Additional context



Discord username (if different from GitHub):

## Testing instructions



## How to download the PR for Testing

Every new commit to this PR will cause a new SU95 artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **HEADWIND-SU95X** download link at the bottom of the page
